end-to-end-tests/specs/home.spec.js

```diff
@@ -0,0 +1,93 @@
+var assert = require('assert');
+
+const CBIOPORTAL_URL = process.env.CBIOPORTAL_URL.replace(/\/$/, "");
+
+describe('homepage', function() {
+
+    it('it should show dev mode when testing', function() {
+        browser.url(CBIOPORTAL_URL);
+
+        browser.localStorage('POST', {key: 'localdev', value: 'true'});
+        browser.refresh();
+
+        var devMode = $('.alert-warning');
+
+        devMode.waitForExist(30000);
```

Comment:
we should make defaultTimeout so we can easily adjust.  there might be one already.  30,0000 seems too high to me.